app/src/main/java/ar/com/wolox/android/example/model/User.java

```diff
@@ -0,0 +1,73 @@
+package ar.com.wolox.android.example.model;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.google.gson.annotations.SerializedName;
+import java.io.Serializable;
+
+/**
+ *
+ */
+public class User implements Serializable {
+
+   @SerializedName("id") private int id;
+   @SerializedName("username") private String username;
+   @SerializedName("email") private String email;
+   @SerializedName("password") private String password;
+   @SerializedName("picture") private String picture;
+   @SerializedName("cover") private String cover;
+   @SerializedName("description") private String description;
+   @SerializedName("location") private String location;
+   @SerializedName("name") private String name;
+   @SerializedName("phone") private String phone;
+
+    public int getId() {
```

Comment:
Here is missing nullability

Reply:
Yes, I don't put nullability here because is a built-in data type **int**, I thought that only have sense to put nullability for objects, for example if it was an **Integer**. Tell me what you think! :smile_cat:

Reply:
Yes, you're absolutely right. If it's a built-in type you don't need a nullability. My bad